controllers/discovery/searchDomain.js

```diff
@@ -20,32 +20,31 @@ const SearchDomain = async (req, res) => {
   try {
     const domains = await sequelize.query(
       `SELECT 
-                "Domain"."domain_page_id",
-                "Domain"."domain_name",
+        "Domain"."domain_page_id" AS "domain_id_followed",
+        "Domain"."domain_name",
+        "Domain"."logo",
+        "Domain"."short_description",
+        "Domain"."credder_score",
+        count("domainFollower"."user_id_follower") AS "followersCount",
+        (
+          SELECT "f"."user_id_follower" AS "user_id_follower" 
+          FROM "user_follow_domain" AS "f" 
+          WHERE "f"."user_id_follower" = :userId 
+          AND "f"."domain_id_followed" = "Domain"."domain_page_id"
+        )
+      FROM "domain_page" AS "Domain" 
+      LEFT JOIN "user_follow_domain" AS "domainFollower" ON "Domain"."domain_page_id" = "domainFollower"."domain_id_followed" 
+      WHERE "Domain"."domain_name" ILIKE :likeQuery 
+      GROUP BY 
+        "Domain"."domain_page_id",
+        "Domain"."domain_name",
 				"Domain"."logo",
 				"Domain"."short_description",
-                "Domain"."credder_score",
-                count("domainFollower"."user_id_follower") 
-                    AS "followersCount",
-                (SELECT "f"."user_id_follower" AS "user_id_follower" FROM "user_follow_domain" AS "f" WHERE "f"."user_id_follower" = :userId AND "f"."domain_id_followed" = "Domain"."domain_page_id")
-            FROM "domain_page" 
-                AS "Domain" 
-            LEFT JOIN "user_follow_domain" 
-                AS "domainFollower" 
-            ON "Domain"."domain_page_id" = 
-                "domainFollower"."domain_id_followed" 
-            WHERE 
-                "Domain"."domain_name" ILIKE :likeQuery 
-            GROUP BY 
-                "Domain"."domain_page_id",
-                "Domain"."domain_name",
-				"Domain"."logo",
-				"Domain"."short_description",
-                "Domain"."credder_score"
-            ORDER BY
-                "user_id_follower" ASC,
-                "followersCount" DESC
-            LIMIT :limit`,
+        "Domain"."credder_score"
+      ORDER BY
+        "user_id_follower" ASC,
+        "followersCount" DESC
+      LIMIT :limit`,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query has been updated to include more columns in the SELECT statement, and the GROUP BY and ORDER BY clauses have been adjusted. However, the ORDER BY clause now includes `"user_id_follower" ASC`, which might not work as expected because "user_id_follower" is not included in the GROUP BY clause. This could potentially cause issues depending on the SQL dialect you're using. Please ensure that this change is intentional and tested.
```diff
- ORDER BY "user_id_follower" ASC,
+ ORDER BY "Domain"."domain_page_id" ASC,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
implement in the new commit